libr/core/cmd.c

```diff
@@ -545,6 +545,10 @@ static int cmd_interpret(void *data, const char *input) {
 		break;
 	case '(':
 		r_cmd_macro_call (&core->rcmd->macro, input+1);
+		free (core->lastcmd);
+		core->lastcmd = malloc (strlen (input) + 2);
+		strcpy (core->lastcmd, ".");
```

Comment:
there's a nullderef here in case malloc fails, use r_str_newf and youll not depend on the strcat which is slow